### DISCUSSION TITLE
Variable not found

### DISCUSSION THREAD

**Discussion by subhash16** [Sketchware-Pro/Sketchware-Pro#782] [Q&A]
![Screenshot_2023-02-24-21-51-28-741_com sketchware remod](https://user-images.githubusercontent.com/37414496/221232203-82a8d271-a26a-4e20-b98d-5544064f56f3.jpg)

**Comment by moderGamer** [comment]
For me, these blocks are not displayed at all `

**Reply by subhash16** [reply]
Place a file on asset you will see

**Comment by j-px** [comment]
@hasrat-ali

**Comment by hasrat-ali** [comment]
That's you bro

**Comment by j-px** [comment]
Didn't you add those Blocks?

**Comment by hasrat-ali** [comment]
Maybe after this commit https://github.com/Sketchware-Pro/Sketchware-Pro/commit/f3e8fda11048d0f843d0b959024c005842cc2060

**Comment by hasrat-ali** [comment]
> Didn't you add those Blocks?

That's Aldi Sayuti

**Comment by j-px** [comment]
> Maybe after this commit [f3e8fda](https://github.com/Sketchware-Pro/Sketchware-Pro/commit/f3e8fda11048d0f843d0b959024c005842cc2060)

How could that be? It's not handling `%m.inputstream` selectors.

**Comment by j-px** [comment]
The "getFileFromAsset path" Block needs you to use the Code Editor option. I'm not sure why that was ever added, but it's already in v6.3.0 fix1, so it won't be removed.

**Reply by subhash16** [reply]
Don't remove any existing block if problem not solved we will use code editor on those blocks to create variable

**Comment by subhash16** [comment]
![Screenshot_2023-02-24-23-20-18-460_com sketchware remod](https://user-images.githubusercontent.com/37414496/221252262-6fbfb013-c18f-47bd-8543-b8f1ce401ba4.jpg)

**Comment by subhash16** [comment]
Menu block also and many more

**Comment by elfilibusterismo** [comment]
Check your folder /.sketchware/resources/block/Menu Block/ if its empty then maybe you've deleted it.

**Reply by elfilibusterismo** [reply]
![Screenshot_2023-02-25-03-12-30-74_9e8df3d0c7c1f50248b6ee043a653d26.jpg](https://user-images.githubusercontent.com/113604229/221270527-70b64da6-f875-4f4d-8519-ff02e72e0c93.jpg)

**Reply by subhash16** [reply]
It's a bug.
I reinstall problem same
